src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,84 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final double LOAD_FACTOR = 0.75;
+    static final int DEFAULT_CAPACITY = 16;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size == table.length * LOAD_FACTOR) {
```

Comment:
is this case better to use `>` or `<` then `==`
```suggestion
        if (size > table.length * LOAD_FACTOR) {
```